scripts/refresh-after-snapshot.py

```
@@ -428,28 +428,54 @@ def pre_fix_cert_sans(config: RefreshConfig) -> None:
+        "apiVersion": "k8s.keycloak.org/v2alpha1",
+        "kind": "KeycloakRealmImport",
+        "metadata": {
+            "name": "osac-realm-import",
+            "namespace": config.keycloak_ns,
+        },
+        "spec": {
+            "keycloakCRName": "osac-keycloak",
+            "realm": realm,
+        },
+    }
     apply_result = subprocess.run(
```

Ruff (0.15.21)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)